public/omni/getting-started/getting-started.mdx

```diff
@@ -18,122 +18,151 @@ If you would like to watch a quick video of the process you can follow along her
 
 ## Prerequisites
 
+You must have the following to create a cluster with Omni:
+
 ### Network access
 
-If your machines have outgoing internet access, you are all set. Machines should have access to the Wireguard Endpoint shown on the Omni Home panel, which lists the IP address and port (e.g. 199.99.99.100:10001) that machines will connect to. Machines need to be able to reach that address both on the UDP port specified, and on TCP port 443.
+Your machines must have outbound internet access and be able to reach the **WireGuard endpoint** shown on your Omni Home panel.  
+This endpoint lists an IP address and port (e.g., `199.99.99.100:10001`) that nodes connect to.
 
-### Create virtual or boot physical machines
+Machines need to reach that address over:
+- **UDP** on the specified port (for WireGuard)
+- **TCP** on port `443` (for Omni communication)
 
-The simplest way to experience Omni is to create some virtual machines. We suggest any virtualization platform that can boot off an ISO (UTM, ProxMox, VMware Fusion, etc) although any cloud platform can also be used with minor adjustments. Bare metal can boot from a physical CD, USB drive, virtual media, or PXE.
+### Virtual or physical machines
 
-### `talosctl`
+You can use any virtualization platform that supports booting from an ISO image, for example, UTM, Proxmox, or VMware Fusion.
+Alternatively, you can deploy on cloud platforms or physical hardware. Bare metal systems can boot using a CD, USB drive, virtual media, or PXE.
 
-`talosctl` is the command line tool for managing Talos Linux via the management API, but when machines connect to Omni it is not required. Instead cluster management is done via the Omni UI or `omnictl`. We still recommend installing `talosctl` to investigate the state of the nodes and explore functionality.
+### CLI tools
```

Comment:
Maybe move this section after the clutser was created where you talked about moving the config files